### ISSUE TITLE
Kyt-starter-static: cannot load styles/scripts from public folder from index.ejs

### ISSUE THREAD

**Issue by Skorney** [NYTimes/kyt#419]
I cannot load static js files from `src/public` folder when using **kyt-starter-static**:

```
// src/index.ejs
<script src="/some-script.js"></script>
<title>Static kyt</title>
```

The file `some-script.js` is inside `src/public` folder.

### Versions of kyt, node, OS

kyt: 0.4.0, node: v7.5.0, windows 10

### Bug or Feature?
Bug

### Current Behavior
I cannot load js scripts from `public` folder from `index.ejs` file.
(404 error at: http://localhost:3001/some-script.js)

### Desired Behavior
I expected it to work the same as in **kyt-starter-universal** where I can load static scripts from `server/template.js` as `<script src="/some-script.js"></script>`.

**Comment by Skorney** [comment]
@delambo `src/public` folder doesn't work for me in **kyt-starter-static**, could you please help my how to proceed with this issue, the direction where to start. I tried to fix it with `publicDir` and `publicPath` options in kyt.config.js but it didn't help.

**Comment by delambo** [comment]
Sorry @Skorney - Give me a day to look into this. I'll get you an answer soon.

**Comment by delambo** [comment]
Hey @Skorney - We got a fix in version 0.4.1. Check it out and let me know if you have any issues.

**Comment by Skorney** [comment]
@delambo 0.4.1 works fine to me. I tested css/js loading from `src/public` folder. Thank you!
